### PR TITLE
Added the Sign-up page.

### LINKED ISSUE #14: [FEATURE] Add Sign-up page.
add sign-up page with its contents.

### PR DESCRIPTION
**Pull request by jaynarayan-vaishnav**
## Short Description
- Added the sign-up page for the website
## Fixes Issue
Closes #14 

## Screenshots
image

## Note to reviewers

this PR is not yet done i have to link the sign-up button to Dashboard page.

### CODE REVIEW

**Comment by jaynarayan-vaishnav** [comment]
@AliReza1083 created a draft PR for now i still have to add the link For buttons to the dashboard page .

**Comment by alisamadiii** [comment]
It is great.

The things if we do, will be awesome: 👇

1. Can we reduce the gap between the buttons?
2. Can we increase the padding-x for each buttons?

**Comment by jaynarayan-vaishnav** [comment]
@AliReza1083  thanks for the review i'll work on it

**Comment by jaynarayan-vaishnav** [comment]
@AliReza1083  is this okay??

image

File: pages/sign-up.js [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+      <section className="flex-col flex gap-6">
+        <Link href="/host">
+          <button
+            type="button"
+            className="rounded-md px-8 py-3 flex gap-2 bg-[#1DA1F2]">
+            <AiOutlineTwitter className="h-6 w-6" />
+            <div>Sign-in as Host</div>
+          </button>
+        </Link>
+        <Link href="/host">
```
Comment (alisamadiii) [inline pages/sign-up.js]: Can we change the href to `speaker` when the user click the button?

File: pages/sign-up.js [hunk 2]
```diff
@@ -1,11 +1,32 @@
+import Link from "next/link";
 import { motion as m } from "framer-motion";
+import { AiOutlineTwitter } from "react-icons/ai";
 
-const singUp = () => {
+const signUp = () => {
```
Comment (alisamadiii) [inline pages/sign-up.js:5]: Can we change the first letter to be Capitalize?

**Review by alisamadiii — CHANGES_REQUESTED** [review]
Rest LGTM, Awesome Work Jay

File: pages/sign-up.js (hunk 2 above)
Comment (jaynarayan-vaishnav) [reply pages/sign-up.js:5]: in javaScript we mostly use camel case for naming thee variables i think

File: pages/sign-up.js (hunk 1 above)
Comment (jaynarayan-vaishnav) [reply pages/sign-up.js]: yup i can do that 👍🏻

Comment (jaynarayan-vaishnav) [reply pages/sign-up.js]: the speaker page is not available rn

Comment (alisamadiii) [reply pages/sign-up.js]: We can add that, No problem.
In the near future, we'll add that page.

Comment (jaynarayan-vaishnav) [reply pages/sign-up.js]: Okay i am changing it for the speakers section thanks for the review.

**Comment by jaynarayan-vaishnav** [comment]
Done For now :D

File: pages/sign-up.js (hunk 2 above)
Comment (alisamadiii) [reply pages/sign-up.js:5]: It is absolutely true but that is for function.
but when we are making a component, We have to use capitalize words, it's a rule.

Comment (alisamadiii) [reply pages/sign-up.js:5]: It's alright.

**Review by alisamadiii — APPROVED** [review]

**Comment by alisamadiii** [comment]
LGTM, Thanks for your Contribution 🎉

**Pull request merged**